staging/src/k8s.io/component-base/cli/globalflag/globalflags_test.go

```diff
@@ -38,27 +35,13 @@ func TestAddGlobalFlags(t *testing.T) {
 		actualFlag = append(actualFlag, flag.Name)
 	})
 
-	// Get all flags from flags.CommandLine, except flag `test.*`.
-	wantedFlag := []string{"help"}
-	pflag.CommandLine.AddGoFlagSet(flag.CommandLine)
-	pflag.VisitAll(func(flag *pflag.Flag) {
-		if !strings.Contains(flag.Name, "test.") {
-			wantedFlag = append(wantedFlag, normalize(flag.Name))
-		}
-	})
-	sort.Strings(wantedFlag)
-
-	if !reflect.DeepEqual(wantedFlag, actualFlag) {
-		t.Errorf("[Default]: expected %+v, got %+v", wantedFlag, actualFlag)
-	}
```

Comment:
This test seemed redundant with the one below, it was removed given that it's testing a behavior that's not true anymore given that we're not registering klog flags globally.